x/crisis/keeper/keeper.go

```diff
@@ -74,8 +74,9 @@ func (k Keeper) AssertInvariants(ctx sdk.Context) {
 
 	start := time.Now()
 	invarRoutes := k.Routes()
-
-	for _, ir := range invarRoutes {
+	fmt.Println("invarRoutes #", len(invarRoutes))
+	for i, ir := range invarRoutes {
+		fmt.Println("invariat nr:", i)
```

Comment:
Ditto

Reply:
sure, I just keep it here if I will need to change anything.